R/optionalInput.R

```diff
@@ -170,6 +170,7 @@ optionalSelectInput <- function(inputId, # nolint
   default_options <- list(
     "actions-box" = multiple,
     "none-selected-text" = "- Nothing selected -",
+    "allow-clear" = ifelse(multiple, FALSE, TRUE),
```

Comment:
`ifelse` is debated a lot recently for several problems. For this simple use case this is equivalent and (to me) is more readable:

```suggestion
    "allow-clear" = !multiple,
```